index.js

```diff
@@ -207,4 +207,3 @@ class Conf {
 }
 
 module.exports = Conf;
-module.exports.default = Conf;
```

Comment:
Is this a breaking change? I'm asking since this is being removed, but the PR title doesn't make it clear.

Reply:
Sorry, I've written this in the description, not in the title. Will change.

Reply:
Oops, I missed that.